README.md

```diff
@@ -62,22 +62,26 @@ Add the following to your Emacs config to enable
 (add-hook 'clojure-mode-hook #'inf-clojure-minor-mode)
 ```
 
-**Don't enable `inf-clojure-minor-mode` and `cider-mode` at the same
-time.  They have overlapping functionality and keybindings and the
-result will be nothing short of havoc.**
+**Don't enable `inf-clojure-minor-mode` and `cider-mode` at the same time. They
+have overlapping functionality and keybindings and the result will be nothing
+short of havoc.**
 
 ## Usage
 
 Just invoke `M-x inf-clojure` or press `C-c C-z` within a Clojure source file.
 This will start a REPL process for the current project and you can start
-interacting with it.
+interacting with it. By defaul this will look for `lein` command on the path.
```

Comment:
This is not accurate. The logic is here, all the projects are tried in order, defaulting to generic: https://github.com/clojure-emacs/inf-clojure/blob/master/inf-clojure.el#L589